### ISSUE TITLE
[Variable Manager] Clearer breaks in the list area.

### ISSUE THREAD

**Issue by LES-11** [ScratchAddons/ScratchAddons#1875]
**Is your feature request related to a problem? Please describe.**

When adding items to a list in the variable manager, it is quite confusing as to which item of the list I am editing.

**Describe the solution you'd like**

- Separate the items of the list with lines (a different color than the lines separating the variables to avoid confusion).
- Number the items of the list on the right hand side.

**Describe alternatives you've considered**

One or both of the options above.

**Additional context**

![Capture](https://user-images.githubusercontent.com/75516401/111093814-0627eb00-84f7-11eb-9271-ea262fd6828c.PNG)

**Comment by WorldLanguages** [comment]
@jeffalo

**Comment by jeffalo** [comment]
yeah i will have to rework how lists work. textareas don't support all the stuff i need
